lib/ui/screens/top_users/top_users_screen.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+
+import '../../../data/models/app_response.dart';
+import '../../../data/models/user.dart';
+import '../../../data/services/firestore/firestore_service.dart';
+import '../../utils/constant.dart';
+
+class TopUserScreen extends StatelessWidget {
+  const TopUserScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(
+      builder: (context, snapshot) {
+        var quarterTurns = 0;
```

Comment:
Why?

Reply:
To rotate the screen when window size minimizes

Reply:
Should we make landscape the default for mobile devices?

Reply:
As of now it is made for Web app, not checked for platform